test/libsolidity/syntaxTests/memberLookup/memory_structs_with_mappings_unused.sol

```diff
@@ -0,0 +1,10 @@
+contract Test {
+	struct S { uint8 a; mapping(uint => uint) b; uint8 c; }
+	S s;
+	function f() public {
+		S memory x;
+		x.a;
+	}
+}
+// ----
+// TypeError: (104-114): Type struct Test.S memory is only valid in storage. Mappings cannot live outside storage.
```

Comment:
This did compile before the change.

Reply:
I think it should still.

Reply:
But what's the use of it? Why should you be able to use all members of the `struct` except the `mapping`. Is there a use case I don't see? Otherwise I think we could fix that weird behavior.

Reply:
Ah, ok. The following does also not compile anymore but I think it should, since the `mapping` member would be valid in the callers storage. 
```
library Test {
    struct Nested { mapping(uint => uint)[2][] a; uint y; }
    struct X { Nested n; }
    function f(X storage x) external {}
}
```

Reply:
yes, that should certainly work.

Reply:
Given https://github.com/ethereum/solidity/issues/6444, maybe it shouldn't after all :-). Although that'd make this breaking. The case in @erak's comment with the library-storage-struct should work in any case, though.